test/dtypes/test_nf4.py

```diff
@@ -270,6 +271,14 @@ def test_chunk_size_equivalence(self, dtype: torch.dtype, shape, chunk_size):
 
         torch.testing.assert_close(nf4_patched.quantized_data, nf4_base.quantized_data)
 
+    @parametrize("input_size", [(512 * 512,), (512, 512)])
+    def test_empty_like(self, input_size: Union[Tuple[int], int]):
+        nf4_tensor = to_nf4(torch.rand(input_size))
+        new_tensor = torch.empty_like(nf4_tensor, device=torch.device("cpu"))
```

Comment:
Not sure why you want to construct device here explicitly
```suggestion
        new_tensor = torch.empty_like(nf4_tensor, device="cpu")
```